test/test_date.rb

```diff
@@ -34,4 +34,16 @@ def test_forward
     assert_random_between(today + 1..today + 30) { @tester.forward(30) }
     assert_instance_of Date, @tester.forward
   end
+
+  def test_birthday
+    today = Date.today
+
+    assert_random_between(today.prev_year(65)..today.prev_year(18)) { @tester.birthday }
+    assert_random_between(today.prev_year(43)..today.prev_year(42)) { @tester.birthday(min_age: 42, max_age: 42) }
+    assert_instance_of Date, @tester.birthday
+
+    leap_day = Date.new(2020, 2, 29)
+    assert_random_between(leap_day.prev_year(65)..leap_day.prev_year(18)) { @tester.birthday }
+    assert_random_between(leap_day.prev_year(43)..leap_day.prev_year(42)) { @tester.birthday(min_age: 42, max_age: 42) }
```

Comment:
I haven't executes it, but.. maybe..
```suggestion
    assert_random_between(leap_day.prev_year(66).next_day..leap_day.prev_year(18)) { @tester.birthday }
    assert_random_between(leap_day.prev_year(43).next_day..leap_day.prev_year(42)) { @tester.birthday(min_age: 42, max_age: 42) }
```

Reply:
I accidentally committed this change. This won't work unless we use TimeCop to change the return value of Date.today in the implementation.